src/Card.js

```diff
@@ -40,63 +41,49 @@ class Card extends Component {
         {reveal}
       </div>
     );
-  }
+  };
 
-  renderAction(actions) {
+  const renderAction = actions => {
     return <div className="card-action">{actions}</div>;
-  }
+  };
 
-  renderContent(title, reveal, textClassName, children) {
+  const renderContent = (title, reveal, textClassName, children) => {
```

Comment:
actually I wonder, could we extract these methods outside of the component definition? since they're all getting values from args

Reply:
But it's using methods that are inside the component.

Reply:
True, i'm just wondering if all these methods receive all the required props as arguments, why not move them outside the component render? .i.e

```jsx
const renderBar = (title) => {
  <p>{title}</p>
}

const Foo = (props) => {
  return (
    <div>
      {renderBar(props.title)}
    </div>
  )
}
```

WDYT?

Reply:
Well, if the function would be reused somewhere else in the project, then yeah, I'd see a point in moving it outside of a component render. But since they belong only to this component I don't really see the advantages, but then again maybe I am missing some details.